Add TestFlight install guide for testers (#542)
nhsuk/nhsapp-frontend: docs/ios/installing-testflight-app.md

---
layout: layouts/ios.njk
title: Installing an app from TestFlight
tags:
  - iosTesting
---

> We use TestFlight to share test versions of the app before they're released. To test the app on your iPhone or iPad, install TestFlight first, then open the invite we've emailed you.

Reviewer: ```suggestion
We use TestFlight to share test versions of the app before they're released.
```

> We use TestFlight to share test versions of the app before they're released. To test the app on your iPhone or iPad, install TestFlight first, then open the invite we've emailed you.

Reviewer 2: Add a link to the TestFlight app?

## Before you start

You'll need:

- an iPhone or iPad
- the email address we used to invite you

> Use the device that's signed in with that email address. Your invite is linked to it, so the app will not install on a device signed in with a different account.

Reviewer 2: ```suggestion
Use the device that's signed in to an Apple ID with that email address. Your invite is linked to it, so the app will not install on a device signed in to an Apple ID with a different account.
```

Make it clear that this is about the Apple ID? (You might be signed in to other things, like email, with other email addresses)

> ## Install TestFlight

Reviewer: ```suggestion
## Step 1: Install TestFlight
```

<div class="nhsuk-inset-text">

>   <p><strong>Important:</strong> Install TestFlight before you open the invite email. If you open the invite first without TestFlight, the link will not work.</p>

Reviewer: I'm not too sure about including 'Important' in inset text, which already draws something out as important. Should we use a warning callout if the inset text doesn't feel strong enough?

</div>

1. On your iPhone or iPad, open the App Store.
2. Search for TestFlight.

> 3. Install TestFlight. It's free and made by Apple.

Reviewer 2: Add link here? Maybe worth keeping the instructions to search though, if you're reading this on a laptop but installing on a phone.

> ## Open your invite email

Reviewer: ```suggestion
## Step 2: Open your invite email
```

1. On the same iPhone or iPad, open the invite email from us.

> 2. Select 'View in TestFlight' (this may say 'Start testing').

Reviewer: ```suggestion
2. Select **View in TestFlight** (this may say **Start testing**).
```

Reviewer: I've just changed some text to bold in the steps, as the service manual formatting guidance is to do this for things we're asking people to select.

3. TestFlight opens automatically.

> ## Accept and install the app

Reviewer: ```suggestion
## Step 3: Accept and install the app
```

> 1. In TestFlight, select 'Accept'.

Reviewer: ```suggestion
1. In TestFlight, select **Accept**.
```

> 2. Select 'Install'.

Reviewer: ```suggestion
2. Select **Install**.
```

3. The app appears on your home screen. Open it from there or from within TestFlight.

## If the invite button does not work

Your invite email also contains a redeem code.

1. Open TestFlight.

> 2. Select 'Redeem'.

Reviewer: ```suggestion
2. Select **Redeem**.
```
